### ISSUE TITLE
All upgrades resetting when downgrading one specific upgrade

### ISSUE THREAD

**Issue by ArefyNetwork** [BG-Software-LLC/SuperiorSkyblock2#1065]
### Minecraft's Version

1.8.8

### Plugin's Version

Latest one, [#595]

### Describe the bug

When I use **is admin setupgrade %player% spawner-rates 2** and after that **is admin setupgrade %player% spawner-rates 1**, players lose their expansion limit and member limits? (basically all their others upgrades)

It happens when you downgrade a upgrade from a player, it resets all upgrades

### To Reproduce

Basically what I've told before

### Additional Information

Look at the expansion how it decreases when decreasing spawner-rate upgrade: https://gyazo.com/767d0bc439d4e2266205d8b44ee33e08

**Comment by OmerBenGera** [comment]
Their upgrade levels are changed as well or the actual limits are changed?

**Comment by ArefyNetwork** [comment]
> Their upgrade levels are changed as well or the actual limits are changed?

What do you mean? For example, they have a 300x300 expansion and 10 members limit (default one is 100x100 and 5 members limit), I give them spawner-rates 2 (for capturing an outpost) and then I downgrade their spawner-rates to 1 (default one), they'll lose their expansion to 100x100 and member limit to 5

**Comment by OmerBenGera** [comment]
> > Their upgrade levels are changed as well or the actual limits are changed?
> 
> What do you mean? For example, they have a 300x300 expansion and 10 members limit (default one is 100x100 and 5 members limit), I give them spawner-rates 2 (for capturing an outpost) and then I downgrade their spawner-rates to 1 (default one), they'll lose their expansion to 100x100 and member limit to 5

The upgrade levels are changing or only the border-size and team limit? You can check it with `/is admin show`

**Comment by OmerBenGera** [comment]
I can't replicate the issue.
Mind uploading your upgrades config file to pastebin and send it here?

**Comment by OmerBenGera** [comment]
Any news regarding this?

**Comment by OmerBenGera** [comment]
Closing this due to lack of response.
If it still occurs, lmk by replying here
